### DISCUSSION TITLE
Temperature input from external source

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#709] [REM] [answered]
I've ordered ORP and PH EZO modules from Atlantic Scientific.  I didn't order a temperature module--maybe I should have, but...

I'm already getting temperature in the screenLogic app.  I can also get it through a hubitat integration that communicates with nodejs poolController.  It's been a year or more since I was into the details, but IIRC it was something like a "body" structure.

Can this be used for the temperature parameter needed for the ORP calculation?  Or, do I need to get another temperature probe?

**Comment by rstrouse** [comment, marked as answer]
You can simply set the checkbox on the REM Chem hardware page and it will feed the water sensor probe data to the EZO-PH.
